src/pages/EventAdministration/components/EventGiftCardSender.tsx

```diff
@@ -0,0 +1,58 @@
+import { PictureAsPdfRounded } from '@mui/icons-material/';
+import { Button } from '@mui/material';
+import { useState } from 'react';
+
+import { useSendGiftCardsToAttendees } from 'hooks/Event';
+import { useSnackbar } from 'hooks/Snackbar';
+
+// Project components
+import FileUploader from 'components/inputs/FileUploader';
+import Dialog from 'components/layout/Dialog';
+
+export type EventFileSenderProps = {
+  eventId: number;
```

Comment:
```suggestion
  eventId: Event['id'];
```

Vet det ikke er gjort mange andre steder, men dette synes jeg er enda mer lesbart